images/hubble-proto/install-protoplugins.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+set -o xtrace
+set -o errexit
+set -o pipefail
+set -o nounset
+
+mkdir /proto
+cd /proto
+
+unset GOPATH
+
+go mod init github.com/cilium/hubble/protoc
+
+go get google.golang.org/grpc/cmd/protoc-gen-go-grpc@8bec2f5d898f
+go build google.golang.org/grpc/cmd/protoc-gen-go-grpc
+
+go get github.com/mitchellh/protoc-gen-go-json@8fbb6f3
+go build github.com/mitchellh/protoc-gen-go-json
```

Comment:
Please add a comment just describing what is `8bec2f` and `8fbb6f3` are pointing to which semver (if possible). Also, if relevant, the reason we are pinning to these commits SHA.

Reply:
Just using what we have right now, actually. No changes here but I can mention why the hashes